### DISCUSSION TITLE
IKEA Tradfri E1810 unsupported

### DISCUSSION THREAD

**Discussion by grozanski** [Koenkk/zigbee2mqtt#12667] [Devices]
Hi,
I’ve got 2 IKEA Tradfri E1810 which are added to Zigbee2mqtt but displaying as “Unsupported”
I’ve got others the same type but they are working correctly.
Any idea what happened here?

Thanks in advance,
Grzegorz

**Comment by netweaver1970** [comment]
old thread but what helped me was delete the device in the Z2M gui and re-pair. Then it appeared and was recognized fine. 
But it doesn't have the _action entity in HA. On the Z2M exposes side, all is fine though. That how I ended up here :)
